src/Servers/IIS/AspNetCoreModuleV2/AspNetCore/AspNetCore.vcxproj

```diff
@@ -1,237 +1,36 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project DefaultTargets="Build" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <ItemGroup Label="ProjectConfigurations">
```

Comment:
Nice❕ Is this file even 20% its previous size❔

Reply:
317 lines to 116, so like 33% but its much saner to make changes to now, it was insane how many copies of compiler flags there were before